src/lib/isMobilePhone.js

```diff
@@ -20,6 +20,7 @@ const phones = {
   'ar-SY': /^(!?(\+?963)|0)?9\d{8}$/,
   'ar-TN': /^(\+?216)?[2459]\d{7}$/,
   'az-AZ': /^(\+994|0)(10|5[015]|7[07]|99)\d{7}$/,
+  'ar-QA': /^(\+?974\s?)?([3567]\d{3})(\s|-)?\d{4}$/,
```

Comment:
```suggestion
  'ar-QA': /^(\+?974|0)?[3567]\d{7}$/,
```
We don't yet have a uniform standard for the regexes, but this should bring it more in line with others. This is an optional suggestion